app/controllers/settings/preferences_controller.rb

```diff
@@ -46,6 +46,7 @@ def user_settings_params
       :setting_noindex,
       :setting_theme,
       :setting_hide_network,
+      :setting_filter_bots,
```

Comment:
There are two approaches to the desired result:

- User setting that affects all queries
- Query param that affects queries like `local`, `only_media`, and web UI setting to store param state

Why did you choose the former?

Reply:
I guess it felt like a similar setting to `chosen_languages`, so I followed a similar path.
I also thought that it would make sense to be able to synchronize this preference with third-party apps.

Reply:
Was...that a request for action? Or just a question?